src/components/Application/SectionSummary.tsx

```diff
@@ -43,7 +43,7 @@ const SectionSummary: React.FC<SectionSummaryProps> = ({
                           isStrictValidation={isStrictValidation}
                         />
                       </Grid.Column>
-                      <Grid.Column floated="right" width={5}>
+                      <Grid.Column floated="right" width={3}>
```

Comment:
I think it's better to leave as `5` just because it was looking better on mobile view

Reply:
Actually, I think it was the `floated` properties that were messing it up. I've removed them now, and it responds properly in mobile view. The float and the Grid were conflicting, I think, and we don't need any float in this situation.

Reply:
It's still not perfect, but I think I need to properly understand how <Grid> works responsively and what all its props mean before going to deep down this rabbit hole.

Reply:
Cool. I had a quick check now and I agree. And anyway we shouldn't make it look great in the mobile view yet. Just the most responsive we can get without much effort. All good!